webrender/src/tiling.rs

```diff
@@ -860,10 +845,10 @@ impl ClipBatcher {
     }
 }
 
-struct RenderTargetContext<'a> {
-    stacking_context_store: &'a [StackingContext],
-    prim_store: &'a PrimitiveStore,
-    resource_cache: &'a ResourceCache,
+pub struct RenderTargetContext<'a> {
```

Comment:
Can we have the fields hidden and only exposed to the module that uses them?
I imagine that both `tiling` and `frame_builder` need to create a `RenderTargetContext`, but do they both need to access the fields?